html/host.html

```diff
@@ -3,66 +3,100 @@
     <meta charset="utf-8"/>
     <link href="/assets/stylesheets/main.css" rel="stylesheet">
     <link href="/assets/stylesheets/host.css" rel="stylesheet">
+    <script type="module">
+      import { 
+        fullActivityDescription, 
+        shortActivityDescription, 
+        libraryModelList, 
+        cclDiseaseActivityUrl 
+      } from '/assets/data/host.js';
+
+      document.addEventListener("DOMContentLoaded", () => {
+        // Host page content init
+        document.getElementById("activity-text-full").innerHTML = fullActivityDescription;
+        document.getElementById("activity-text-short").innerHTML = shortActivityDescription;
+        document.getElementById("library-model").innerHTML = libraryModelList;
+        document.getElementById("activity-img").src = cclDiseaseActivityUrl;
+
+        // Open & close modal functionality
+        const readMoreBtn = document.getElementById("read-more-btn");
+        const closeModalBtn = document.getElementById("close-modal-btn");
```

Comment:
These abbreviations of "button" upset me.  I would approve of writing it out as "button".  I'm very big on explicit clarity.  And I have lots of great arbitrary rules, just like this one.

Reply:
Not arbitrary at all & totally easy to fix! Done!